### DISCUSSION TITLE
怎么藏手机啊

### DISCUSSION THREAD

**Discussion by liuyaozhu** [share121/inter-knot#1623] [其他]
哥们难受坏了，天天晚上收手机
![Uploading 20240809113451.jpg…]()

**Comment by suiry-D** [comment]
放在奶粉铁罐里。不仅防肉眼寻找，还防信号检测。

**Reply by liuyaozhu** [reply]
感谢

**Comment by hzjwork** [comment]
行李箱里，天花板能打开就天花板，空调外机，床板地下，柜子夹缝里，空调里面，都是我高中用过的，不过最好还是要把精力放在学习上

**Reply by hzjwork** [reply]
实在不行可以把两个塑料盆叠在一起，中间夹手机

**Reply by liuyaozhu** [reply]
感谢

**Comment by liuyaozhu** [comment]
我是初中生，在我姥姥家住，晚上不让我玩绝区零

---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月12日(周一) 下午2:08
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] 怎么藏手机啊 (Discussion #1623)




 
行李箱里，天花板能打开就天花板，空调外机，床板地下，柜子夹缝里，空调里面，都是我高中用过的，不过最好还是要把精力放在学习上

**Reply by canifish** [reply]
你和你姥姥不是一个房间睡吧？如果你父母不反对的话你就和你姥姥说开让她别管那么紧，不过还是要早点睡啊

**Comment by canifish** [comment]
看你想防谁，防家长买备用机，藏他们已经知道的手机很难藏，防学校容易点，行李箱或者柜子上锁，或者下铺床板用牛奶盒粘上去形成一个刚好可以放手机的小空间，别带去教室就行，影响学习又容易挨抓

**Comment by liuyaozhu** [comment]
不是，谢谢你



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年8月12日(周一) 下午2:16
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [share121/inter-knot] 怎么藏手机啊 (Discussion #1623)




 
你和你姥姥不是一个房间睡吧？如果你父母不反对的话你就和你姥姥说开让她别管那么紧，不过还是要早点睡啊
